mmdet3d/models/voxel_encoders/pillar_encoder.py

```diff
@@ -47,14 +46,14 @@ def __init__(self,
                  point_cloud_range=(0, -40, -3, 70.4, 40, 1),
                  norm_cfg=dict(type='BN1d', eps=1e-3, momentum=0.01),
                  mode='max',
-                 legacy=True):
```

Comment:
This changes default behavior. Should we avoid BC-breaking?

Reply:
Double check, there is no need for modifications here.